lib/kotkowo/gallery_image.ex

```diff
@@ -1,7 +1,8 @@
 defmodule Kotkowo.GalleryImage do
   @moduledoc false
   defstruct [:hash, :ext, :name, :alternative_text]
-
+  # idk add smthn here xddd
+  @default_image "placeholder"
```

Comment:
This should be an actual image.
Add an image to `priv/static/images/FILENAME` and set this to `/images/FILENAME`.
It can be an image of a cat. Remember to take an image with CC0 license